modules/mapper-extras/src/test/java/org/elasticsearch/index/query/RankFeatureQueryBuilderTests.java

```diff
@@ -52,7 +52,7 @@ protected void initializeAdditionalMappings(MapperService mapperService) throws
 
     @Override
     protected Collection<Class<? extends Plugin>> getPlugins() {
-        return Collections.singleton(MapperExtrasPlugin.class);
+        return List.of(MapperExtrasPlugin.class, TestGeoShapeFieldMapperPlugin.class);
```

Comment:
I think there is room to pull the `TestGeoShapeFieldMapperPlugin` to the parent test class and inherited by those that need it or need to extend to include more plugins. I could see it stay either way since it is fairly low impact and the dependency is expected to go away